content/en/docs/tasks/configure-pod-container/subpath-env-expansion.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Config the subpath using environment variables
+content_template: templates/task
+---
+
+{{% capture overview %}}
+This page shows how to configure a subpath and allow expansion of a variable host path
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* The feature gate VolumeSubpathEnvExpansion must be enabled
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Using kubectl to create a pod definition
+
+
+* The host path can be mounted into the pod and written to as any other volume
```

Comment:
Always spell Pod as 'Pod', first character capitalized.

Reply:
Instead of saying 'host path', you may want to instead using the field names in the yaml file.
BTW, this feature is only applicable to hostpath volumes?